cscs-checks/prgenv/helloworld.py

```diff
@@ -26,10 +26,11 @@ def __init__(self, variant, lang, linkage):
         self.valid_systems = ['daint:gpu', 'daint:mc', 'dom:gpu', 'dom:mc',
                               'arolla:cn', 'arolla:pn', 'tsa:cn', 'tsa:pn']
         if linkage == 'dynamic':
-            self.valid_systems.append('eiger:mc')
+            self.valid_systems += ['eiger:mc', 'pilatus:mc']
 
-        self.valid_prog_environs = ['PrgEnv-cray', 'PrgEnv-cray_classic',
-                                    'PrgEnv-intel', 'PrgEnv-gnu', 'PrgEnv-pgi',
+        self.valid_prog_environs = ['PrgEnv-aocc', 'PrgEnv-cray',
+                                    'PrgEnv-cray_classic', 'PrgEnv-gnu',
+                                    'PrgEnv-intel', 'PrgEnv-pgi',
```

Comment:
For the moment the collection work on Eiger and Pilatus, with the exception of `PrgEnv-intel` only (failing on Pilatus). 
Since module collections for `PrgEnv-xxx` modules will be dropped by Cray as of PE 21.04, it won't be an issue.
Anyway, `Lmod` provides the environment variable `$LMOD_SYSTEM_NAME` that can be uniquely defined for each system to avoid conflicts: see https://lmod.readthedocs.io/en/latest/010_user.html#user-collections-on-shared-home-file-systems

Reply:
Can we try this ?
https://lmod.readthedocs.io/en/latest/010_user.html?highlight=lmod.d#user-collections-on-shared-home-file-systems